web-ui/src/components/member_selector/member_selector_dialog/MemberSelectorDialog.jsx

```diff
@@ -1,14 +1,20 @@
+import { differenceInMonths } from 'date-fns';
```

Comment:
This function is used to determine if a member has been at OCI for at least the selected tenure value.